layouts/states/single.html

```diff
@@ -0,0 +1,18 @@
+{{ partial "head.html" . }}
+
+  {{ $registration_type := .Params.registration_type }}
+
+  {{ $registration_template := printf "register-%s" $registration_type }}
+
+  <body class="page-register-{{ .Params.registration_type }}">
+    <div id="main" role="main">
+      <a class="skipnav" href="#main-content">Skip main navigation</a>
+
+      {{ .Render $registration_template }}
```

Comment:
The `states/single.html` layout calls the other registration templates as needed based on the `registration_type` within the content page found in `content/states/<NAME>.md`.